translations/strings.yaml

```diff
@@ -562,6 +562,7 @@ tipsAndTricks:
   tutorialsDescription: Learn the basics of Mozilla VPN
   moreTutorialsTitle: More tutorials
   moreTutorialsDescription: Check out some of our other interactive tutorials
```

Comment:
Is this still used? If not, it should be removed.

Reply:
```suggestion
```